### PR TITLE
Code review: website accessibility 

### PR DESCRIPTION
**Pull request by NasKhalil**
This branch includes the following features implemented:

Updated HTML file named (index.html)

accessibility report:

1. Page Titles - No Issues found.
2. Headings - Changed the h1 tags to ensure there is only one h1 tag in the documents.
3. Image text alternatives - Changed and added more detailed descriptions of the images in alternative attribute
4. Color contrast - no issues found
5. Resize - no issues found
6. Interaction - no issues found
7. Moving content -no issues found
8. Multimedia - Website has no multimedia elements.
9. The basic structure of the page -no issues found
10. update the form labels display

### CODE REVIEW

**Review by NasKhalil — COMMENTED** [review]
Commit id( 12ab141 ): I made a mistake, I did not pull the code.

File: index.html [hunk 1]
```diff
@@ -30,8 +30,7 @@
     </header>
```
Comment (SajjadAhmad14) [inline index.html:30]: Please write a detailed accessibility report here in the Pull request summary.

The report should include a check about page titles.
The report should include a check about image text alternatives ().
The report should include a check about text headings.
The report should include a check about color contrast.
The report should include a check about resizing.
The report should include a check about interaction.
The report should include a check about moving content.
The report should incldue a check about multimedia.
The report should include a check about the basic structure of the page.
In the result section of the report the student should include a statement pertained to every point checked:
  If everything is ok, please make a statement about that (e.g. "Page titles - no issues found").
  If issues are found, please add a link to the corresponding line of code in the Pull Request changes.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Sam, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -30,8 +30,7 @@
...
     <main>
         <section class="headline" id="intro">
-            <h1>I'm Khalil</h1>
-            <h1>Glad to see you!</h1>
+            <h1>I'm Khalil <br> Glad to see you!</h1>
```
Comment (NasKhalil) [inline index.html:33]: Changed the h1 tags to ensure there is only one h1 tag in the documents.
update heading h1

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>I'm Khalil <br> Glad to see you!</h1>
         </section>
         <section class="work" id="my-project">
             <div class="container">
-                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-1.png" alt="snapshout portfolio">
+                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-1.png" alt="Screenshot of the Tonic project">
```
Comment (NasKhalil) [inline index.html:50]: Changed and added more detailed descriptions of the images in the alternative attributes.

File: index.html [hunk 4]
```diff
@@ -67,7 +67,7 @@ <h3 class="year">2015</h3>
                 </div>
             </div>
             <div class="container-reverse">
-                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-2.png" alt="snapshout portfolio">
+                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-2.png" alt="Screenshot of the Multi-Post Stories project">
```
Comment (Mosams) [inline index.html:70]: Changed and added more detailed descriptions of the images in the alternative attributes.

File: index.html [hunk 5]
```diff
@@ -107,7 +107,7 @@ <h3 class="year">2015</h3>
                 </div>
             </div>
             <div class="container-reverse">
-                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-4.png" alt="snapshout portfolio" >
+                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-4.png" alt="Screenshot of the Multi-Post Stories project" >
```
Comment (Mosams) [inline index.html:110]: Changed and added more detailed descriptions of the images in the alternative attributes.

File: index.html [hunk 6]
```diff
@@ -123,7 +123,7 @@ <h3 class="year">2015</h3>
                         <li class="tag-2 tag-style">css</li>
                         <li class="tag-3 tag-style">javascript</li>
                     </ul>
-                    <button class="project-button">See Project</button>
+                    <button class="project-button" type="button">See Project</button>
```
Comment (Mosams) [inline index.html:126]: Added the type attribute to the button element.

File: style.css [hunk 7]
```diff
@@ -674,6 +674,7 @@ label {
     display: block;
     color: #fff;
     margin: 10px 0 -5px 0;
+    text-align: start;
   }
```
Comment (Mosams) [inline style.css]: 1. Updated the form labels display which were initially only placeholder. This is to help with accessibility through website text to voice conversion software. 
2. Also changed text color from black to while to improve readability of labels and contrast.

**Review by dasileker — APPROVED** [review]
## **Hi @NasKhalil,** 

### **Good Job**

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**